app/renderer/reducers/frameReducer.js

```diff
@@ -128,7 +128,14 @@ const frameReducer = (state, action, immutableAction) => {
           }
           state = state.setIn(['frames', index, 'lastAccessedTime'], new Date().getTime())
 
-          state = frameStateUtil.updateTabPageIndex(state, frame)
+          // Only update tabPage if user is not hovering other page
+          // Otherwise each time the active tab is updated the tabPage
+          // will be updated as well. While we want that behavior not necessarily
+          // the user will be on the same tabPage as the active tab. See bug #9779
+          const isPreviewing = state.get('frames').some(frame => frame.get('hoverState'))
```

Comment:
actually is it even necessary to clearn `previewTabPageIndex` inside `updateTabPageIndex`? I think it's covered by hoverTabPageIndex now. Maybe we can just remove it? The same probably applies to `previewFrameKey` above